os/arch/arm/src/imxrt/imxrt_pwm.c

```diff
@@ -956,7 +996,39 @@ static int imxrt_pwm_shutdown(FAR struct pwm_lowerhalf_s *dev)
 static int imxrt_pwm_ioctl(FAR struct pwm_lowerhalf_s *dev, int cmd,
 						 unsigned long arg)
 {
-	return -ENOTTY;
+	FAR struct imxrt_pwmtimer_s *priv = (FAR struct imxrt_pwmtimer_s *)dev;
+	int ret = -ENOTTY;
+
+	switch (cmd) {
+	case PWMIOC_IDLE:
```

Comment:
Could you please let me know what is different if user call below ?
fd = open("/dev/pwm");
ioctl(fd, PWMIOC_SETCHARACTERISTICS)
ioctl(fd, PWMIOC_START) ?